app/Console/Commands/SendTaskReminders.php

```diff
@@ -29,9 +29,7 @@ class SendTaskReminders extends Command
     public function handle()
     {
         $now = Carbon::now();
-        $upcomingTasks = Task::where('reminder_time', '<=', $now)
-            ->where('reminder_time', '>=', $now->clone()->subMinutes(10))
-            ->get();
+        $upcomingTasks = Task::where('last_notification_date', null)->get();
```

Comment:
```suggestion
        $upcomingTasks = Task::where('last_notification_date', null)
            ->where('reminder_time', '>=', $now->clone()->subMinutes(10))
            ->get();
```
You still need to filter for notification to send in the next 10 minutes. Otherwise this is going to send an email immediately after the task is created.

Reply:
True. will do